GOVERNANCE.md

```diff
@@ -0,0 +1,46 @@
+
+## Your organization is currently a maintainer
+
+* First decide whether your organization really needs more people with maintainer access. Valid reasons are "blast radius", a large organization that is working on multiple unrelated projects, etc.
+* Contact a senior maintainer for your organization and express interest.
+* Start doing PRs and code reviews under the guidance of your senior maintainer.
+* After a period of 1-2 months the existing senior maintainers will discuss granting "standard" maintainer access. This will require 3 (senior) maintainers +1 vote and no -1 vote.
+* "Standard" maintainer access can be upgraded to "senior" maintainer access after another 1-2 months of work and another conference of the existing senior committers.
+
+## Maintainer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community.
+* Triage build issues - file issues for known flaky builds or bugs, and either fix or find someone to fix any master build breakages.
+* During GitHub issue triage, apply all applicable [labels](https://github.com/lyft/amundsen/labels) to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply is somewhat subjective so just use your best judgment.
+* Make sure that ongoing PRs are moving forward at the right pace or closing them.
+* In general continue to be willing to spend at least 25% of ones time working on Amundsen (~1.25) business days per week).
+
+
+## When does a maintainer lose maintainer status
+
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the maintainers per the voting process below.
+
+
+## Conflict resolution and voting
+
+In general, we prefer that technical issues and maintainer membership are amicably worked out between the persons involved. If a dispute cannot be decided independently, the maintainers can be called in to decide an issue. If the maintainers themselves cannot decide an issue, the issue will be resolved by voting. The voting process is a simple majority in which each senior maintainer receives two votes and each normal maintainer receives one vote (we could introduce senior maintainer like PMC later).
+
+## Adding new projects to the amundsen-io GitHub organization
+
+New projects will be added to the amundsen-io organization via GitHub issue discussion in one of the existing projects in the organization. Once sufficient discussion has taken place (~3-5 business days but depending on the volume of conversation), the maintainers of *the project where the issue was opened* (since different projects in the organization may have different maintainers) will decide whether the new project should be added. See the section above on voting if the maintainers
```

Comment:
```suggestion
We will add new projects to the Amundsen organization via GitHub issue discussion in one of the organization's existing projects. Once sufficient discussion has taken place (~3-5 business days but depending on the volume of conversation), the maintainers of *the project where the issue was opened* (since different projects in the organization may have different maintainers) will decide whether the new project should be added. See the section above on voting if the maintainers
```

Reply:
Not clear what `new projects` here refers to. Do you mean adding `new features to Amundsen` and creating an issue in one of the sub-components?

Reply:
I think this means new repositories on the organization.